src/Mizar/Mizar.cpp

```diff
@@ -53,24 +53,30 @@ int main(int argc, char* argv[]) {
   orbit_mizar_data::BaselineAndComparison bac =
       CreateBaselineAndComparison(std::move(baseline), std::move(comparison));
 
-  constexpr uint64_t kStart = 0;
+  constexpr uint64_t kStart = 20'000'000'000;
```

Comment:
?

Reply:
Here we throw out the first 20s of the capture, that correspond for initialisation of the benchmark I am playing with. The current main function is just for experimenting and will be heavily changed. I'll add a comment.